core/new-gui/src/app/dashboard/user/component/user-workflow/user-workflow-list-item/user-workflow-list-item.component.html

```diff
@@ -4,7 +4,6 @@
     <nz-list-item-meta-avatar [ngStyle]="{ 'margin-left': editable ? 'none' : '16px' }">
       <label
         *ngIf="editable"
-        nz-checkbox
```

Comment:
why do we remove the checkbox?

Reply:
It's invalid attribute and does nothing here.

Reply:
maybe it is because it should be an `input` rather than a `label`?